.env.template

```diff
@@ -0,0 +1,10 @@
+# Flask server configuration.
+FLASK_APP=app
+FLASK_ENV=development
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+KEY = '62ce4878a6a08e0d91f50f1de7ef29be'
+TOKEN = 'fe926cd3836c9d391b11d756c4445b78da11e145d38d89c5ee133fa35ab5a5b7'
+LIST_ID = '5f516c55b4b1d670902e4e36'
+DONE_LIST_ID = '5f51742dc2cdc729993a8914'
```

Comment:
You've put your credentials in github! These values should all be blank in .env.template or `KEY=change me to your Trello account key` or similar.

Also, FYI the .env file is not Python. So technically you should not have those spaces around the `=` signs. The python-dotenv library can handle what you've written, but some parsers will fail

Reply:
done both